modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbPartitionStorage.java

```diff
@@ -120,7 +123,7 @@ private static short shortPartitionId(int intValue) {
     /**
      * Starts the storage.
      *
-     * @throws IgniteInternalException In case when the operation has failed.
+     * @throws StorageException In case when the operation has failed.
```

Comment:
I suggest making an exception for module `ignite-transactions`, say `TxStorageException`, and throwing it.